src/UI/Component/Layout/Page/Factory.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace ILIAS\UI\Component\Layout\Page;
+/**
+ * This is the factory for page-layout components
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The cockpit is a unique page section that holds globally available
```

Comment:
We switched to "navigational elements and globally available tools" and excluded modification.